tox.ini

```diff
@@ -15,15 +15,10 @@ passenv =
   TRAVIS_JOB_NUMBER
   TRAVIS_PULL_REQUEST
   TRAVIS_REPO_SLUG
-deps =
-    pytest
```

Comment:
`pytest` is missing from `project.optional-dependencies.test`. This is why CI is failing.

Reply:
Thanks. It looks like we still need to define all dependencies in tox.ini or make it run pip install independently. Not sure, which way is worse. For now I optioned for using commands_pre to run pip install .[test].

Reply:
Here you're overriding dependencies, so they need to be in the `deps =` array (e.g.: these two are not extras).

Fixing this tests work.